aten/src/ATen/native/LinearAlgebraUtils.h

```diff
@@ -278,4 +278,31 @@ static inline Tensor same_stride_to(const Tensor& original_tensor, const at::Ten
   return strided_to;
 }
 
+// Creates a dimension permutation array that can be given to at::permute(), which will shift
+// the two specified dimensions to the end of a tensor
+static inline std::vector<int64_t> create_dim_backshift_permutation(int64_t dim0, int64_t dim1, int64_t ndim) {
```

Comment:
Better extend this comment, maybe with a simple example. My understanding is it takes an input like

```
1, 3, 4
```

And this creates a new vector with the same order for the dimensions that aren't selected, but it places the two selected dimensions at the end. So:

```
vec(0, 2, 1, 3,)
```

Is that correct?